packages/engine/Source/Scene/LabelCollection.js

```diff
@@ -118,8 +118,74 @@ function addGlyphToTextureAtlas(textureAtlas, id, canvas, glyphTextureInfo) {
 
 const splitter = new GraphemeSplitter();
 
+// Filter to remove unsupported control characters like RLM (\u200f)
```

Comment:
Can you explain a bit about how this list was determined? Would it make sense to test for [a unicode range](https://stackoverflow.com/a/51602415) instead of listing each character individually?

Reply:
see my comment please

Reply:
Any thoughts on using the unicode ranges to make this more concise? I think the following regex matches of these values:

`/[\u0000-\u001F\u202a-\u206f\u200b-\u200f]/`

Reply:
got it